packages/console/src/pages/Dashboard/index.tsx

```diff
@@ -23,35 +33,102 @@ const Dashboard = () => {
               </div>
+            </Card>
+            <Card className={styles.block}>
+              <div className={styles.title}>{t('dashboard.new_users_7_days')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{newData.last7Days.count}</div>
+                <div
+                  className={classNames(styles.delta, newData.last7Days.delta < 0 && styles.down)}
+                >
+                  ({newData.last7Days.delta > 0 && '+'}
+                  {newData.last7Days.delta})
+                  {newData.last7Days.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                </div>
+              </div>
+            </Card>
+          </div>
+          <Card>
+            <div className={styles.dau}>
+              <div className={styles.title}>{t('dashboard.daily_active_users')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{activeData.dau.count}</div>
+                <div className={classNames(styles.delta, activeData.dau.delta < 0 && styles.down)}>
+                  ({activeData.dau.delta > 0 && '+'}
+                  {activeData.dau.delta}){activeData.dau.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                </div>
+              </div>
+            </div>
+            <div className={styles.curve}>
+              <ResponsiveContainer>
+                <AreaChart
+                  data={activeData.dauCurve.map((item) => ({
+                    ...item,
+                    // Remove "year" for a compact label.
+                    date: item.date.replace(/\d{4}-/, ''),
+                  }))}
+                  width={1100}
+                  height={168}
+                >
+                  <CartesianGrid vertical={false} />
+                  <Area
+                    type="monotone"
+                    dataKey="count"
+                    stroke="#5D34F2"
+                    strokeWidth={2}
+                    fill="#F2EFFD"
+                  />
+                  <XAxis dataKey="date" />
+                  <YAxis orientation="right" axisLine={false} tickLine={false} />
+                  <Tooltip />
+                </AreaChart>
+              </ResponsiveContainer>
+            </div>
+            <div className={styles.activeBlocks}>
+              <Card className={styles.block}>
+                <div className={styles.title}>{t('dashboard.weekly_active_users')}</div>
+                <div className={styles.content}>
+                  <div className={styles.number}>{activeData.wau.count}</div>
+                  <div
+                    className={classNames(styles.delta, activeData.wau.delta < 0 && styles.down)}
+                  >
+                    ({activeData.wau.delta > 0 && '+'}
+                    {activeData.wau.delta}){activeData.wau.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                  </div>
+                </div>
+              </Card>
+              <Card className={styles.block}>
```

Comment:
These cards look similar, consider extracting it into a component

Reply:
refactored